apps/wallet/src/ui/app/staking/stake/UnstakeForm.tsx

```diff
@@ -209,6 +209,16 @@ export function UnStakeForm({ stakedIotaId, validatorAddress, epoch, onSuccess }
                     />
                 </div>
             )}
+            {!isLoading && isError && (
```

Comment:
are you sure isError is always when you dont have enough gas? I dont think this is accurate

Reply:
Also, lets better use messages we already use in the app for these cases instead of harcoding them https://github.com/iotaledger/iota/blob/99e937af083bc96d518586480f9d4bc31a6088ba/apps/core/src/constants/errorMessages.constants.ts#L7

Reply:
for the error i think  you have to try matching the error message or something to make sure you are showing the right thing, otherwise for any error you display the not enough gas message https://github.com/iotaledger/iota/blob/99e937af083bc96d518586480f9d4bc31a6088ba/apps/wallet-dashboard/app/(protected)/(feature-controlled)/vesting/page.tsx#L180

Reply:
the usage of GAS_BUDGET_ERROR_MESSAGES applies to the dashboard too